src/SourceBrowser/src/BinLogToSln/Program.cs

```diff
@@ -339,6 +352,142 @@ void includeFile(string originalPath, out string projectRelativePath, out string
                         File.Copy(filePath, outputFile);
                     }
                 }
+
+                void includeReference(string kind, string path)
+                {
+                    if (!File.Exists(path))
+                    {
+                        Console.WriteLine($"Could not find {kind} '{path}'");
+                        return;
+                    }
+
+                    string projToRepoPath = Path.GetRelativePath(invocation.ProjectDirectory, repoRoot);
+                    string projToOutputPath = Path.Join(projToRepoPath, "..");
+                    string refPath = DedupeReference(output, path);
+                    project.WriteLine($"    <{kind} Include=\"{Path.Join(projToOutputPath, refPath)}\"/>");
+                }
+
+#nullable enable
+                // From https://github.com/jaredpar/complog/blob/a629fb3c05e40ebe673410144e8911bd5f86bdf2/src/Basic.CompilerLog.Util/RoslynUtil.cs#L440.

```

Comment:
I've also considered using Basic.CompilerLog.Util directly here, but currently this project is using more information (properties of the MSBuild evaluated project) than the complog library makes available.

Reply:
> What all properties are being used here?

`UseForSourceIndex`, `IsPlatformNotSupportedAssembly`, `TargetFramework`. But I guess it doesn't matter which specific properties are being used, it would be best to have the complog util expose all project properties like the BinLogParser utility in this repo does:

https://github.com/dotnet/source-indexer/blob/07dd0cc31b3c696edb9252302a878f98ea795819/src/SourceBrowser/src/BinLogParser/BinLogReader.cs#L213

> Also should we just make that method in CompilerLog public?

That would be good, yes, I will take a look at that.